lib/features/family/shared/design/components/actions/fun_button.dart

```diff
@@ -179,6 +180,7 @@ class FunButton extends StatelessWidget {
         Text(
           text,
           textAlign: TextAlign.center,
+          overflow: TextOverflow.ellipsis,
```

Comment:
Ellipsis works quite terrible in flutter. Cuts out the entire word sometimes resulting in very weird gaps. Have you tested this change? Not saying that we have a good alternative but sometimes thats why I use clip/fade/visible instead.